The Deathsquadening

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This redoes the descriptions of all deathsquad related items save like, the ID. Removes the NT documents from THE CLAW, buffs deathsquad hardsuits by giving them 80 in laser and energy (to match the other stats), and changes the death commando outfits slightly.
The deathsquad now actually has their own subshuttle.
![image](https://github.com/shiptest-ss13/Shiptest/assets/55075747/622bfd3b-2377-42d2-808d-fd286d7854bc)
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The deathsquad were NT associated prior to this. We want them to be associated with a comic series and be dubiously real. They now belong to an unnamed shadow organization of undescribed scope and power, with the best gear available. Because the dsquad exist to be the strongest unit in game for admins to let players clean up their messes with guns.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:phoaly
add: reflavored the dsquad entirely
add: watchman-class subshuttle
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
Total Deathsquad conversion

- this rewrites every description referencing the deathsquad.
- buffs the dsquad armor for energy and laser to 80 to match their other stats
- removes the nanotrasen documents from claw

Latentish committed:
Total Deathsquad conversion

- replaces centcom uniform with black jumpsuit
- changes death_commando/officer to death_commando/shielded, renamed it to Death Commando Solo, and makes it change the hardsuit to the shielded version instead of giving them a weird cc beret

Latentish committed:
whoopsie

- forgor :skull: to change the associated helmet path smile

Latentish committed:
Minor fixes

- grammatical error fixes
- fixes the outfit

Latentish committed:
couple things

- adds the framework for the watchman

Latentish committed:
Finishes the watchman

- finishes the deathsquad subshuttle map

Latentish committed:
adds the watchman to the catalogue

- titular

Latentish committed:
grah

- adds the datum for the subshuttle

Latentish committed:
Redoes some cabling

- pass the damn checks

Latentish committed:
Merge branch 'master' into The-REDACTED]Squad-(death)

meemofcourse commented on code/datums/ert.dm:
```diff
@@ -49,7 +49,7 @@
 	code = "Delta"
 	mission = "Leave no witnesses."
-	polldesc = "an elite Nanotrasen Strike Team"
+	polldesc = "an overwhelmingly powerful and mysterious Strike Team"
```
```suggestion
	polldesc = "a Death Commando Team"
```
tone it back girl this is supposed to be the default for what the ghosts are seeing

meemofcourse commented on code/modules/clothing/outfits/standard.dm:
```diff
@@ -158,7 +158,7 @@
 	name = "Death Commando"
 
-	uniform = /obj/item/clothing/under/rank/centcom/commander
+	uniform = /obj/item/clothing/under/color/black
```
wouldn't it be better to have a combat uniform here instead of a black jumpsuit

meemofcourse commented on code/modules/projectiles/guns/energy/pulse.dm:
```diff
@@ -29,7 +29,7 @@
 /obj/item/gun/energy/pulse/carbine
 	name = "pulse carbine"
-	desc = "A next-generation pulse weapon for Nanotrasen's security forces. High production costs and logistical issues have limited its deployment to specialist Loss Prevention and Emergency Response units."
+	desc = "A next-generation pulse carbine capable of decimating overwhelming opposition. This weapon has accrued a kill count of 2564 dead since it was first seen in issue #10 'DEATHSQUAD CLASSICS'."
```
an exact kill count will probably be outdated real fast

rye-rice replied:
can you make it change a var and said var is displayed in the description

Latentish replied:
> can you make it change a var and said var is displayed in the description

No because it's talking about the kc in the comics. But I'll change this.

Latentish replied:
changed

meemofcourse commented on _maps/shuttles/subshuttles/Subshuttle Catalog.txt:
```diff
@@ -78,4 +78,7 @@ Size = "15x11"
 File Path = "_maps\shuttles\subshuttles\frontiersmen_brawler.dmm"
 
-
+Name = "Watchman Final Solution"
```
please. please change the name

Latentish replied:
yeah this was pretty bad

meemofcourse reviewed (commented):
![imagen](https://github.com/user-attachments/assets/cc79ce19-7bc0-49f1-949b-bb1e823b8d95)
this seems like a bad idea

Latentish committed:
Update code/datums/ert.dm

Co-authored-by: meem <75212565+meemofcourse@users.noreply.github.com>
Signed-off-by: Latency <55075747+Latentish@users.noreply.github.com>

Latentish committed:
requested changes

- changes the shuttle name
- changes the pulse carbine description
- changes the undershirt for the dsquad is now the military combat uniform.
- removes explosive walls

meemofcourse reviewed (approved):
i will change the outfits to work with the job system later but i think this is good

meemofcourse commented on code/datums/shuttles.dm:
```diff
@@ -411,3 +411,8 @@
+
+/datum/map_template/shuttle/subshuttles/watchman
+	file_name = "deathsquad_watchman"
+	name = "Watchman-class Final Solution"
```
```suggestion
	name = "Watchman Devastator"
```

Latentish committed:
Merge branch 'master' into The-REDACTED]Squad-(death)

Latentish committed:
Update code/datums/shuttles.dm

woopsies

Co-authored-by: meem <75212565+meemofcourse@users.noreply.github.com>
Signed-off-by: Latency <55075747+Latentish@users.noreply.github.com>

Gristlebee reviewed (commented):
So at a first glance:

- You're missing pictures of this ship in strongdmm with areas, and ingame.
- When spawned in game the ship flies backwards.
-  There appears to be atmos differences in the ship on spawn, which throws everything around on spawn.
- Holofields are not anchored.
- The widebands overlay oddly when the ship is rotated vertically.
![image](https://github.com/user-attachments/assets/0bd8a68d-8bbe-49cb-b732-ebf46f06c3a7)
- Same with the railing over the lockers.
![image](https://github.com/user-attachments/assets/db72aea9-261b-4ed3-9504-cf1318709863)
- The bomb casing doesn't fit in the locker
![image](https://github.com/user-attachments/assets/edbcd86f-00c1-4ce2-b297-742558d61301)
- the manifold here spawns under the tile unlike the rest of pipes, making it look inconsistent.  
![image](https://github.com/user-attachments/assets/1f9766d2-1df8-43b2-bd3e-4afeaec4d893)
- One side of the wing is smoothed, while the other is angled.
![image](https://github.com/user-attachments/assets/ec76fa81-b15a-4e0c-bef3-42c4737e5580)

 
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

thgvr commented:
Re-open this if you're coming back to it.
